templates/java/org/prism/Nodes.java.erb

```diff
@@ -174,7 +174,9 @@ public abstract class Nodes {
 <%# NODES -%>
     <%- nodes.each do |node| -%>
 
-    <%= "#{node.comment.split("\n").map { |line| "// #{line}" }.join("\n    ")}\n" if node.comment -%>
+    <%- node.comment.to_s.each_line do |line| -%>
+    #<%= line.rjust(line.size + 1).rstrip %>
+    <%- end -%>
```

Comment:
The best in Java would be to use a JavaDoc comment `/** ... * ... */` like templates/javascript/src/nodes.js.erb below

Reply:
Thanks, done.